src/dosbox.cpp

```diff
@@ -507,12 +507,15 @@ void DOSBOX_Init(void) {
 	pstring->Set_values(force);
 
 #if C_OPENGL
-	Pstring = secprop->Add_path("glshader", Property::Changeable::Always, "sharp");
-	Pstring->Set_help("Path to GLSL shader source to use with OpenGL output (\"none\" to disable).\n"
-	                  "Can be either an absolute path, a file in the \"glshaders\" subdirectory\n"
-	                  "of the DOSBox configuration directory, or one of the built-in shaders:\n"
-	                  "advinterp2x, advinterp3x, advmame2x, advmame3x, rgb2x, rgb3x, scan2x,\n"
-	                  "scan3x, tv2x, tv3x, sharp.");
+	pstring = secprop->Add_path("glshader", always, "sharp");
+	pstring->Set_help(
+	        "Path to GLSL shader source to use with OpenGL output\n"
+	        "(\"none\" to disable).  Can be either an absolute path,\n"
+	        "a file in the \"glshaders\" subdirectory of the DOSBox\n"
+	        "configuration directory, or one of the built-in shaders:\n"
+	        "advinterp2x, advinterp3x, advmame2x, advmame3x,\n"
+	        "crt-easymode-flat, crt-fakelottes-flat, rgb2x, rgb3x,\n"
```

Comment:
Did some testing with these, and feel we need to add some explanation/recommendations for the CRT shaders. Perhaps beneath the list? 

- CRT shaders simulate an analog CRT on 1080p and 4k LCD-panels
- `crt-fakelottes-flat` and `output = opengl` should be used full screen on 1080p displays
- `crt-easymode-flat` and `output = openglnb or pp` should be used on 4k displays
- The effect of both CRT shaders diminishes at game resolutions above 640x480

Reply:
They both work just fine on my 1366x768, and I think on 720p screen as well - of course, I bet they look much better on high-resolution screens (especially in text modes), but they look *fine* on my old laptop :)

They start to look crappy at 800x600 or smaller resolution - the code I added for autodetecting window size for `windowresolution=default` should prevent that from happening unless user wants to :)

I see no reason for confusing users and recommending different output modes per shader - I see no difference, example:

`opengl`:
![opengl-Screenshot from 2020-12-02 16-26-30](https://user-images.githubusercontent.com/3967/100893473-b1b6ad80-34bb-11eb-8e2d-2ca89f604e39.png)

`openglnb`:
![opnglnb-Screenshot from 2020-12-02 16-27-07](https://user-images.githubusercontent.com/3967/100893519-bbd8ac00-34bb-11eb-9a9f-aa778db15e95.png)

`openglpp`:
![openglpp-Screenshot from 2020-12-02 16-28-08](https://user-images.githubusercontent.com/3967/100893619-d90d7a80-34bb-11eb-98ea-c6487367f78d.png)

Reply:
Oh - that's good news then!

All of my screenshots in Wayland are broken (fullscreen mode the surface is empty; window-mode the images are super-dar).. so I wasn't able to to do post-gameplay comparisons.

Let's drop those two suggestions.

Reply:
Perhaps we should still mention: `CRT shaders simulate an analog CRT on LCD-panels`?